.github/workflows/release.yaml

```diff
@@ -22,7 +22,9 @@ env:
   PACKAGE_NAME: {{project-name}}
   # The itch.io page to upload to, in the format: `user-name/project-name`.
   # Comment this out to disable.
+  {%- if itch_username != "" -%}
   ITCH_TARGET: {{itch_username}}/{{project-name}}
+  {%- endif -%}
```

Comment:
Would it be possible to make this line `#ITCH_TARGET:` when no username is provided, so that there's a clear place to add the itch info if the user chooses to do so later on?

Reply:
(redundant with Jan's comment)